src/Targets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -34,5 +34,9 @@
     <PropertyPageSchema Include="$(CustomCommonXamlResourcesDirectory)None.xaml">
       <Context>File;BrowseObject</Context>
     </PropertyPageSchema>
+
+    <PropertyPageSchema Include="$(CustomCommonXamlResourcesDirectory)ConfigurationGeneral.xaml">
```

Comment:
Good catch, will add to that also

---
In reply to: [69232578](https://github.com/dotnet/roslyn-project-system/pull/283#discussion_r69232578) [](ancestors = 69232578)